explore-analyze/dashboards.md

```diff
@@ -9,14 +10,42 @@ products:
 
-Dashboards are the best way to visualize and share insights from your {{es}} data.
+**Dashboards** provide the primary way to visualize and share insights from your {{es}} data. Build interactive displays that combine multiple visualizations, metrics, and controls into a single view that helps you and your team monitor trends, investigate issues, and make data-driven decisions.
 
-A **dashboard** is made of one or more **panels** that you can organize as you like. Each panel can display various types of content: **visualizations** such as charts, tables, metrics, and maps, **static annotations** like text or images, or even **specialized views** for Machine Learning or Observability data.
+A dashboard is composed of one or more **panels** that you arrange to tell your data story. Each panel can display visualizations such as charts, tables, metrics, and maps, static annotations like text or images, or specialized views for Machine Learning or Observability data.
 
 ![Example dashboard](/explore-analyze/images/kibana-dashboard-overview.png "")
 
-At any time, you can [share a dashboard](dashboards/sharing.md) you’ve created with your team, in {{kib}} or outside.
+## Get started with dashboards [get-started]
 
-Some dashboards are created and managed by the system, and are identified as `managed` in your list of dashboards. This generally happens when you set up an integration to add data. You can’t edit managed dashboards directly, but you can [duplicate](dashboards/duplicate-dashboards.md) them and edit these duplicates.
+New to dashboards? Start with these hands-on tutorials that walk you through creating your first dashboards using sample data:
+
+- [Create a simple dashboard to monitor website logs](dashboards/create-dashboard-of-panels-with-web-server-data.md): Build a dashboard with charts, metrics, and filters to analyze web traffic patterns
+- [Create a dashboard with time series charts](dashboards/create-dashboard-of-panels-with-ecommerce-data.md): Learn to visualize sales trends and patterns over time using ecommerce data
+
+
+## Work with dashboards [work-with-dashboards]
+
+Once you understand the basics, explore these common tasks:
+
+**Build and customize dashboards**:
+- [Building dashboards](dashboards/building.md): Learn the fundamentals of creating and configuring dashboards
+- [Create a dashboard](dashboards/create-dashboard.md): Start with an empty dashboard and add your content
+- [Add filter controls](dashboards/add-controls.md): Enable interactive filtering with options lists, range sliders, and time sliders
+- [Add drilldowns](dashboards/drilldowns.md): Create interactive navigation between dashboards or to external URLs
+- [Organize dashboard panels](dashboards/arrange-panels.md): Arrange panels using collapsible sections, resizing, and positioning
+
+**Explore and interact**:
+- [Exploring dashboards](dashboards/using.md): Filter data, adjust time ranges, and interact with panels to uncover insights
```

Comment:
```suggestion
- [Explore dashboards](dashboards/using.md): Filter data, adjust time ranges, and interact with panels to uncover insights
```
This would match the sentence structure from the preceding list, but the following list using more "ing"s, so I'm not sure which way you want to go, but I think we should pick one.